packages/client/hmi-client/src/components/model-template/tera-model-template.vue

```diff
@@ -18,7 +18,8 @@
 				</header>
 				<tera-model-diagram :model="model" :is-editable="isEditable" is-preview />
 			</main>
-			<Button v-if="isEditable" icon="pi pi-ellipsis-v" rounded text />
+			<Button v-if="isEditable" @click="toggle" icon="pi pi-ellipsis-v" rounded text />
+			<Menu ref="menu" :model="cardOptions" :popup="true" />
```

Comment:
[In the PrimeVue documentation](https://primevue.org/menu/#popup) you can see that they use the `aria-` attributes to do the job of your `toggle` function. That would be cleaner.

Reply:
No the toggle function is actually still there. You gotta toggle the full code 😛